read-fonts/src/tables/sbix.rs

```diff
@@ -22,3 +22,41 @@ impl<'a> Strike<'a> {
         Ok(Some(GlyphData::read(data)?))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{FontRef, TableProvider};
+
+    #[test]
+    fn sbix_strikes_count_overflow() {
+        // Contains an invalid `num_strikes` values which would move the cursor outside the able.
+        let test_case = &[
```

Comment:
just for informational purposes, there is a `test_helpers` module in this crate that contains a `BeBuffer` type that is intended for creating intelligible test cases for this sort of scenario; using that we could write a version of this that looked like,

```rust
#[test]
fn sbix_strikes_count_overflow() {
    // Contains an invalid `num_strikes` values which would move the cursor outside the able.
    let sbix = BeBuffer::new()
        .push(1u16) // version
        .push(0u16) // flags
        .push(u32::MAX);

    let table = Sbix::read(sbix.font_data(), 5);
    table.unwrap();
}
```

which makes it easier for the reader to understand the problem.

(I haven't checked that this example actually fails)

Reply:
Thanks for the suggestion and distilling the test case - indeed, this works (same, on 32-bit) only, but is indeed a smaller and more legible way to express that. I was expecting to run into trouble trying to make other tables align with sbix for this to trigger but using the `read_with_args` entry point makes this easier, thanks for demoing that.